smartsim/entity/model.py

```diff
@@ -291,7 +303,39 @@ def _set_colocated_db_settings(
         if hasattr(self.run_settings, "_prep_colocated_db"):
             self.run_settings._prep_colocated_db(common_options["cpus"])
 
+        if "limit_app_cpus" in common_options:
+            raise SSUnsupportedError(
+                "Pinning of app CPUs via limit_app_cpus is no supported. Modify RunSettings " +
```

Comment:
```py
"Pinning of app CPUs via limit_app_cpus is no supported. Modify RunSettings " +
#                                          ^^ no -> not
```
Also style thing: The trailing `+` is not strictly needed for concatenation of two str literals, but it gets optimized away at the byte code level, so feel free to keep or leave depending on your own personal style